src/main/java/com/salesforce/comdagen/model/Site.kt

```diff
@@ -27,6 +27,10 @@ class Site(
     val regions
         get() = config.regions
 
+    val id: String
+        // Delete all non alphanumeric characters in siteName.
+        get() =  if(config.siteName != null) config.siteName.replace("[^A-Za-z0-9]".toRegex(),"") else "Site_" + internID.toString()
```

Comment:
Please use official [formatting](http://kotlinlang.org/docs/reference/coding-conventions.html#formatting), there's also an IntelliJ automatic formatter you can apply. In this case, just lacking a bunch of whitespace.